src/routes/activityReports/handlers.js

```diff
@@ -157,6 +157,32 @@ export async function resetToDraft(req, res) {
   }
 }
 
+/**
+ * Mark activity report status as deleted
+ *
+ * @param {*} req - request
+ * @param {*} res - response
+ */
+export async function softDeleteReport(req, res) {
+  try {
+    const { activityReportId } = req.params;
+
+    const report = await activityReportById(activityReportId);
+    const user = await userById(req.session.userId);
+    const authorization = new ActivityReport(user, report);
```

Comment:
I was a little confused the first time I read this because of the overloading of this class name, which happens in other parts of the application. I wonder if we might be better off with more distinctive class names, or in the case of the activity report policy, not make it a class at all? `canDelete(user, report)` is easier for me to understand.

Reply:
I agree that overloading the class name makes for a confusing read. However, this is how all the activity report policies are written (and this is the 13th policy.)  Let's refactor in a new PR.

Reply:
Yes, totally something for a new PR!